addon/components/hyper-table-v2/filtering-renderers/common/facets-loader.ts

```diff
@@ -97,7 +97,11 @@ export default class HyperTableV2FacetsLoader extends Component<FacetsLoaderArgs
   private fetchFacets(): void {
     this.loading = true;
     this.args.handler
-      .fetchFacets(this.args.column.definition.key, this.args.facettingKey, this.searchQuery)
+      .fetchFacets(
+        this.args.column.definition.key,
+        this.args.column.definition.facetable_by[0] || 'value',
```

Comment:
How does this behave when `facetable_by` is null?  As it is it might raise an error ?

Reply:
Was thinking it would default to an empty array on your side if nothing was passed in it. But I'll add an extra check then 🙂

Reply:
Read: I'd rather be able to deploy this now so that we have no issues when QAing